### DISCUSSION TITLE
Firmware

### DISCUSSION THREAD

**Discussion by Abd-Bin** [azerimaker/Penguino-STM32WL-LoRa-E5#5] [General]
Hi,

Any idea from where I can get the sample projects for STM32WL.? I have a custom designed development board.

Regards!

**Comment by azerimaker** [comment]
Hi, 

@Abd-Bin . If you install the official tool from ST, the STM32CubeIDE, there are a lot of example projects already. 

Hope it helps. 

Cheers!
